Services/DataX.Flow/DataX.Flow.InteractiveQuery/Databricks/DatabricksKernel.cs

```diff
@@ -58,6 +58,11 @@ public string ExecuteCode(string code)
             var request = new HttpRequestMessage(HttpMethod.Post, url) { Content = new FormUrlEncodedContent(nvc) };
             var response = client.SendAsync(request).Result;
             var responseString = response.Content.ReadAsStringAsync().Result;
+            if (!response.IsSuccessStatusCode)
+            {
+                //If request fails then return error string
+                return responseString;
```

Comment:
Returning a raw response may not be appropriate as it may contain uncomprehending xml. Instread return only the error code and the message.

Reply:
In this context the response message displays the actual error message, for eg.: '{"error":"ContextNotFound: "}' . While the error code only says 500 and response phrase says 'Internal server error' without giving detail about the error. Thus returning the response